common/script/content.coffee

```diff
@@ -1919,7 +1919,7 @@ api.subscriptionBlocks =
   basic_12mo: months:12, price:48
 _.each api.subscriptionBlocks, (b,k)->b.key = k
 
-repeat = {m:true,t:true,w:true,th:true,f:true,s:true,su:true}
+# repeat = {m:true,t:true,w:true,th:true,f:true,s:true,su:true}
```

Comment:
Ah, sorry, I actually misread your first comment.

Yes, good point about not needed.